crates/dprint/src/environment/environment.rs

```diff
@@ -19,6 +19,9 @@ pub trait Environment: Clone + std::marker::Send + std::marker::Sync + 'static {
   fn mk_dir_all(&self, path: impl AsRef<Path>) -> Result<(), ErrBox>;
   fn cwd(&self) -> PathBuf;
   fn log(&self, text: &str);
+  fn log_verbose(&self, text: &str) {
+    eprintln!("{}", text);
```

Comment:
Maybe skip the `take_logged_errors()` change. I'm doing a bit of a refactor right now to support verbose logging in the vscode extension/editor service and there will be many conflicts. I'll just rename it as part of my refactoring.

Reply:
Sounds good!  cc @ryan-rushton for the verbose logging in the editor service

Reply:
Heh, I actually ended up forgetting to rename those methods in my refactor 😅. Feel free to submit a PR if you would like, but otherwise I will get to it soon.

The editor service and other changes are in the latest 0.16.1 release. I'm just going to do a vscode update now.